lib/app/getPackageFile.js

```diff
@@ -32,19 +32,19 @@ module.exports = function (config) {
 // our list of dependencies for each
 var deps = {
     express: {
-        "body-parser": "^1.4.3",
-        "compression": "^1.0.8",
-        "cookie-parser": "^1.3.2",
-        "express": "^4.6.1",
-        "helmet": "^0.3.2",
-        "jade": "^1.3.1",
-        "moonboots-express": "^0.1.1",
-        "semi-static": "^0.0.5",
-        "serve-static": "^1.3.2"
+        'body-parser': '^1.4.3',
+        'compression': '^1.0.8',
+        'cookie-parser': '^1.3.2',
+        'express': '^4.6.1',
+        'helmet': '^0.3.2',
+        'jade': '^1.3.1',
+        'moonboots-express': '^0.1.1',
+        'semi-static': '^0.0.5',
```

Comment:
Should we update some of these if they are off by major versions? The ones I see that won't get the latest based on the `^` semver range are:

- helmet (0.7.1)
- moonboots-express (3.0.0)